web/pages/resources.tsx

```diff
@@ -0,0 +1,489 @@
+      },
+      {
+        email: "catadopt",
+        purpose: "for cat adoption questions and issues",
+      },
+      {
+        email: "catpetfinder",
+        purpose:
+          "to send pictures, a bio, and video of your cat foster for adoption sites",
+      },
+      {
+        email: "catpost",
+        purpose:
+          "to have your foster cat posted on the main page to help with adoption",
+      },
+      {
+        email: "catadopted",
+        purpose:
+          "to turn in adoption paperwork once your foster cat is adopted",
+      },
+    ],
+  },
+  {
+    teamName: "Volunteer Teams",
+    primaryContact: " Susan DuBois (susan@angelsrescue.org)",
+    contacts: [
+      {
+        email: "volunteers",
+        purpose:
+          "to offer to volunteer on a team or to transport, general volunteering questions",
+      },
+      {
+        email: "recognition",
+        purpose:
+          "to ask that a card be sent to another foster or volunteer going through a difficult time or for going above and beyond",
+      },
+      {
+        email: "fbgroups",
+        purpose:
+          "to be added to one of our internal pages, must be current foster or volunteer",
+      },
+    ],
+  },
+  {
+    teamName: "Social Media & Marketing Teams",
+    primaryContact: "Hilary Van Brunt (hilaryv@angelsrescue.org)",
+    contacts: [
+      {
+        email: "socialmedia",
+        purpose:
+          "to send pictures and video of your foster for posting on our social medial platforms",
+      },
+      {
+        email: "photography",
+        purpose:
+          "to have professional pictures taken of your foster by our volunteer photographers",
+      },
+      {
+        email: "marketing",
+        purpose:
+          "for marketing opportunities and needs, including flyers, brochures, etc.",
+      },
+    ],
+  },
+  {
+    teamName: "Development & Events Teams",
+    primaryContact: "Jackie Spett (jackie@angelsrescue.org)",
+    contacts: [
+      {
+        email: "events",
+        purpose:
+          "for questions about Angels events or to direct people asking about having an event",
+      },
+      {
+        email: "matchinggifts",
+        purpose: "if your company does matching gifts",
+      },
+      {
+        email: "donate",
+        purpose:
+          "donations questions, problems, or to request a thank you card",
+      },
+    ],
+  },
+];
+
+interface TeamProps {
+  team: Team;
+  children: JSX.Element | JSX.Element[];
+}
+
+interface TeamContactProps {
+  contact: string;
+  purpose: string;
+}
+
+function TeamContact({ contact, purpose }: TeamContactProps) {
+  return (
+    <Flex
+      bgColor="tag-primary-bg"
+      flexDir="column"
+      paddingX={4}
+      paddingY={3}
+      borderRadius={10}
+    >
+      <Text fontWeight="600">{`${contact}@angelsrescue.org`}</Text>
+      <Text>{purpose}</Text>
+    </Flex>
+  );
+}
+
+function Team({ team, children }: TeamProps) {
```

Comment:
Use `PropsWithChildren<T>` instead of manually typing children - modify `TeamProps` to just include `team`

Reply:
Wait that's crazy